### PR TITLE
Test enhance

### PR DESCRIPTION
**Pull request by BJSherman80**
Description:  Add more robust testing to test the category of each call is correct according to the uri
* 

Fixes # 

* [x] All tests passing/ except on travis we have the test marked out. 
* [x] Lint code (most errors exist in serializer which can't be fixed)
* [x] Checked local host
* [x] Updated README

### CODE REVIEW

File: spec/application_controller_spec.rb [hunk 1]
```diff
@@ -94,6 +94,247 @@ def app
...
+    found = false
+    data = JSON.parse(last_response.body, symbolize_names: true)
+    data = data[:data]
+    data.each do |biz|
+      found = true if biz[:categories].include?('Electricity Suppliers')
+    end
+    expect(found).to eq(true)
+  end
+
+  xit "can hit the utilities/isps endpoint" do
```
Comment (Stearnzy) [inline spec/application_controller_spec.rb]: Weren't these unskipped in Jake's PR?

**Review by Stearnzy — COMMENTED** [review]
I'm confused by the differences in your and Jakes PR

File: spec/application_controller_spec.rb (hunk 1 above)
Comment (Oxalisviolacea) [reply spec/application_controller_spec.rb]: I second this.

**Review by Oxalisviolacea — APPROVED** [review]

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
-require './config/environment'
-require 'sinatra'
+# require './config/environment'
+# require 'sinatra'
```
Comment (hopesgit) [inline app/controllers/application_controller.rb]: Why are these commented out?

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -14,118 +16,106 @@ class ApplicationController < Sinatra::Base
 
   # do we want to change this to '/:utility/:location' so that we only need one get request? This would have to pass in the utility they want (e.g. electricity) as well as what needs to be passed to yelp (e.g. electricitysuppliers)
```
Comment (hopesgit) [inline app/controllers/application_controller.rb:13]: Left a comment in

File: config/environment.rb [hunk 4]
```diff
@@ -3,8 +3,8 @@
 require "bundler"
 Bundler.require(:default, ENV['SINATRA_ENV'])
 Bundler.require(Sinatra::Base.environment)
-require "active_support/deprecation"
-require "active_support/all"
+# require "active_support/deprecation"
+# require "active_support/all"
```
Comment (hopesgit) [inline config/environment.rb]: Same as before, are these supposed to be commented out?

File: spec/application_controller_spec.rb [hunk 5]
```diff
@@ -107,7 +348,7 @@ def app
       expect(biz).to have_key(:name)
       expect(biz[:name]).to be_an(String)
       expect(biz).to have_key(:is_closed)
-      expect(biz[:is_closed]).to be_in([true, false])
+      expect(biz[:is_closed]).to be(true).or be(false)
```
Comment (hopesgit) [inline spec/application_controller_spec.rb:351]: I like this, this is clever

**Review by jakeheft — APPROVED** [review]

**Pull request merged**